users/tests/test_permissions.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from django.contrib.auth.hashers import make_password
+
+from core.models import Community, CommunityMember, Post
+from users.models import User
+
+
+@pytest.fixture()
+def user() -> User:
+    return User.objects.create_user(email="user@example.com", nickname="user", password=make_password("password"))
+
+
+@pytest.fixture()
+def user_without_post() -> User:
+    return User.objects.create_user(
+        email="user2@example.com",
+        nickname="user_without_task",
+        password=make_password("password"),
+    )
+
+
+@pytest.fixture()
+def moderator() -> User:
+    return User.objects.create_user(
+        email="moderator@example.com",
+        nickname="moderator",
+        password=make_password("password"),
+    )
+
+
+@pytest.fixture()
+def admin() -> User:
+    return User.objects.create_user(email="admin@example.com", nickname="admin", password=make_password("password"))
+
+
+@pytest.fixture()
+def community(user: User) -> Community:
```

Comment:
not needed, please use 
https://github.com/pymasterspl/reddit/blob/6810df3c26114963e2be001cad26822025886179/conftest.py#L55-L56